docker/Dockerfile.oasisui_app

```diff
@@ -7,6 +7,10 @@ FROM rocker/r-ver:3.5.1
 RUN apt-get update && \
     apt-get install -y --no-install-recommends libv8-3.14-dev libcairo2-dev libxt-dev libcurl4-gnutls-dev libssl-dev libssh2-1-dev libxml2-dev apt-utils && \
     # if desired, add tools such as: sudo, vim, wget, curl, procps
+    # library necessary for dependency of geojsonio
+    apt-get install -y libprotobuf-dev protobuf-compiler  && \
+    # library necessary for dependency of geojsonio
+    apt-get install libjq-dev  && \
```

Comment:
This would have to be with the `-y` option as well. But I would actually suggest to include it in the line above, add the `--no-install-recommends` option too and re-test. I believe we miss a few more dependencies like `gdal-bin`, `proj-bin`, `libgdal-dev` and possibly `libudunits2-dev` (not sure exactly about debian stretch).

Reply:
added with commit  efc34b6